packages/manager/apps/hub/src/dashboard/dashboard.controller.js

```diff
@@ -1,5 +1,22 @@
+import { DOMAINS_REPRACING, PRODUCT_TYPE_DOMAIN } from './dashboard.constants';
+
 export default class DashboardController {
   $onInit() {
+    this.hasDomainRepricing = false;
+
+    // Check if the tld of domains are present in the list affected by the domain repricing MANAGER-9696.
```

Comment:
**question:** I think we dont mention the ticket numbers in the codebase. Could you please check and correct if required?

Reply:
This was to highlight the ephemeral nature of the motion.